src/ray/protobuf/export_api/export_train_run_info.proto

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option cc_enable_arenas = true;
+
+package ray.rpc;
+
+message ExportTrainRunInfo {
+  // State of an train run.
+  enum RunStatus {
```

Comment:
do we need an UNKNOWN enum?

I think it's a pattern of protobufs: https://stackoverflow.com/a/17164524

Reply:
Yeah, this is a common best practice. The other export API schemas don't follow this, but I'll update those in a followup